tests/testthat/test_standardise_dates.R

```diff
@@ -50,48 +46,48 @@ test_that("standardise_dates() treats reverse ordered dates correctly", {
 
 test_that("standardise_dates() treats future dates correctly",{
   expect_warning(recent(), "is deprecated")
-  expect_match(as.character(standardise_dates("9999-12-31")), "9999-12-31")
-  expect_match(as.character(standardise_dates("2599-12-31")), "9999-12-31")
+  expect_match(standardise_dates("9999-12-31"), messydates::as_messydate("9999-12-31"))
+  # expect_match(standardise_dates("2599-12-31"), messydates::as_messydate("9999-12-31"))
```

Comment:
Please remove all the tests that are not pertinent to the new standardise_dates() function instead of commenting them out.